actor/src/main/scala/com/evolutiongaming/akkaeffect/util/Serially.scala

```diff
@@ -44,27 +57,33 @@ private[akkaeffect] object Serially {
       def apply(f: A => F[A]) = {
         for {
           d <- Concurrent[F].deferred[Either[Throwable, Unit]]
-          t  = (a: A) => {
+          t = (a: A) => {
             for {
               b <- f(a).attempt
               _ <- d.complete(b.void)
             } yield {
               b.getOrElse(a)
             }
           }
-          a <- ref.modify {
-            case s: S.Idle   =>
-              (S.Active, start(s.value, t))
-            case s: S.Active =>
-              val task = (a: A) => Async[F].defer {
-                for {
-                  a <- s.task(a)
-                  a <- t(a)
-                } yield a
-              }
-              (S.Active(task), Concurrent[F].unit)
-            case S.Active    =>
-              (S.Active(t), Concurrent[F].unit)
+          a <- Sync[F].delay {
```

Comment:
no need to write everything in `Sync[F].delay {`

Reply:
Fixed